model_training/local_prediction/README.md

```diff
@@ -0,0 +1,46 @@
+# Improving Model using LabelImg
+
+## Prerequisites
+
+Tested on
+
+- Windows 10 (not WSL, since `cv2.imshow` is being used to visualise predictions)
+- Python 3.7
+- [LabelImg](https://github.com/tzutalin/labelImg/releases)
+
+## Getting Started
+
+1. Create and enter `virtualenv` if not done so already.
+   ```bash
+   $ virtualenv venv
+   $ .\venv\Scripts\activate
+   ```
+1. Install `opencv`.
+   ```bash
+   $ pip install opencv-python
+   ```
```

Comment:
We can utilize the `Pipfile` and `Pipfile.lock`.
Try running:
```
cd local_prediction
pipenv install
pipenv shell
python yolo_object_detection.py
```

This is a better virtual environment manager than python-venv